.azure-pipelines/Linux-CI.yml

```diff
@@ -61,8 +61,8 @@ jobs:
       fi
       export CMAKE_ARGS="-DONNX_WERROR=ON -DONNX_USE_PROTOBUF_SHARED_LIBS=ON"
       export ONNX_NAMESPACE=ONNX_NAMESPACE_FOO_BAR_FOR_CI
-      # enable UndefinedBehaviorSanitizer
-      export CMAKE_ARGS="${CMAKE_ARGS} -DCMAKE_CXX_FLAGS='-fsanitize=undefined,address -fno-sanitize-recover=undefined '"
+      # enable more sanitizer
+      export CMAKE_ARGS="${CMAKE_ARGS} -DCMAKE_CXX_FLAGS='-fsanitize=undefined,address,null,alignment -fno-sanitize-recover=undefined '"
```

Comment:
I think null is part of undefined?

Reply:
> I think null is part of undefined?

Thanks. I think you are right.

BTW, currently the added sanitizer checks look fine without error when building ONNX. Is there something missing from the config? If not, is it possible that actually https://github.com/onnx/onnx/blob/10d126ef0754e32aeba93d714444fd0ab72d1e7f/onnx/defs/tensor_proto_util.cc#L129 is OK from ONNX perspective?

Either way, adding more sanitizer checks should be good to have.

Reply:
The code will need to be executed I think. Was the line covered?

Reply:
This needs to be `-fno-sanitize-recover=all ` it seems: https://gcc.gnu.org/onlinedocs/gcc/Instrumentation-Options.html#:~:text=%2Dfsanitize%2Drecover%5B%3Dopts%5D